src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -88,23 +88,22 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * Saves a given manual textAssessment
-     * TODO SE: refactor this REST call to not use the exerciseId anymore, and make uniform with other save..Assessment calls
-     * @param exerciseId the exerciseId of the exercise which will be saved
+     * PUT text-submissions/result/:resultId/assessment : Saves a given manual textAssessment
+     *
      * @param resultId the resultId the assessment belongs to
      * @param textAssessment the assessments
      * @return 200 Ok if successful with the corresponding result as body, but sensitive information are filtered out
      */
-    @PutMapping("/exercise/{exerciseId}/result/{resultId}")
+    @PutMapping("text-submissions/result/{resultId}/assessment")
```

Comment:
I don't think this URL makes sense. Either there is as submissionId in between

1) `text-submissions/{submissionId}/result/{resultId}/assessment`

or we leave it out

2) `results/{resultId}/assessment`

Might be easier to work with option 2).

In general and for the future, I would prefer if we have one element for the authorization check in all URLs, either `courseId`, `exerciseId` or `participationId`, but this could also be refactored in a follow-up.